### PR TITLE
migrate code from googleapis/nodejs-document-ai

### PR DESCRIPTION
**Pull request by ace-n**
- chore: initial build of document ai library
- feat!: initial generation of beta client (#5)
- feat(samples): adds Document AI parse table, forms samples (#8)
- feat: updated library from protos (#9)
- feat: adds beta samples
- chore: release 1.0.0 (#6)
- chore: removes unneeded samples (#25)
- feat!: drop node8 support (#30)
- chore(deps): update dependency @types/mocha (#18)
- chore: update region tags (#33)
- fix(samples): do not pass location (#39)
- chore: fix samples according to service resource change (#35)
- fix(deps): update dependency @google-cloud/storage to v5 (#56)
- chore: release 2.0.0 (#31)
- chore(deps): update dependency mocha to v8 (#63)
- chore: release 2.1.0 (#61)
- chore: release 2.1.1 (#75)
- build(test): recursively find test files; fail on unsupported dependency versions (#100)
- feat: generation error and set v1beta3 as default (#112)
- docs(samples): new Doc AI samples for v1beta3 (#101)
- chore: release 2.2.0 (#106)
- chore: release 2.2.1 (#128)
- docs: make projectId variable consistent in quickstart.js comments
- chore: release 2.2.2 (#132)
- docs: make processor ID variable consistent with comments in snippet (#136)
- chore: release 2.3.0 (#144)
- chore: release 2.4.0 (#153)
- chore: release 2.5.0 (#155)
- chore: release 2.6.0 (#182)
- chore: release 2.7.0 (#193)
- chore: release 2.8.0 (#197)
- samples: updates v1beta3 to v1; renames v1beta2 samples (#198)
- chore: release 2.9.0 (#205)
- chore: new owl bot post processor docker image (#212)
- chore: release 2.10.0 (#213)
- samples: Update quickstart.js (#189)
- chore: release 3.0.0 (#221)
- chore: release 3.0.1 (#244)
- chore: release 3.0.2 (#254)
- chore: release 3.1.0 (#259)
- docs(samples): add OCR, form, quality, splitter and specialized processing samples (#283)
- samples: added default value for undefined (#287)
- chore: release 3.2.0 (#288)
- chore(main): release 3.3.0 (#306)
- chore: removed v1beta2 samples (#325)
- build!: update library to use Node 12 (#333)
- fix(deps): update dependency @google-cloud/storage to v6 (#335)
- chore(main): release 4.0.0 (#334)
- chore(main): release 4.1.0 (#340)
- chore(main): release 5.0.0 (#344)
- fix(deps): update dependency uuid to v9 (#356)
- chore(main): release 6.0.0 (#347)
- chore(main): release 6.1.0 (#374)

### CODE REVIEW

File: .github/workflows/workflows.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
   "datalabeling",
   "datastore/functions",
   "datacatalog/quickstart",
+  "document-ai",
```
Comment (NimJay) [inline .github/workflows/workflows.json:32]: **Praise:** Thanks for redo-ing this PR and removing the `document-ai/snippets/` sub-folder.

**Review by NimJay — APPROVED** [review]
The file changes look good to me.
Approved.
I have overridden the CLA check. ✅ 
Ping me if you need another override. :)

**Comment by ace-n** [comment]
@NimJay thanks! I can CLA override too, but I'd like a Node reviewer (other than myself - @grayside maybe?) to sign off here. 🙂

File: document-ai/quickstart.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
+/**
```
[the remark this answers is not in the document]
Comment (ace-n) [reply document-ai/quickstart.js:1]: Deleting, per @NimJay's suggestion. 👍

**Review by grayside — APPROVED** [review]

**Comment by ace-n** [comment]
- `ci/docs` failure is unrelated
- CLAs look good

➡️ time to **admin merge**.

**Pull request merged**